src/configparser.js

```diff
@@ -231,22 +237,39 @@ ConfigParser.prototype.writeAsync = function(file) {
     return writeFileAsync(file, out);
 }
 
-function parseLines(lines) {
+function parseLines(lines, options) {

```

Comment:
we could just use default params, so `function parseLines(lines, options = {}) {...}`